modules/understanding-quick-start-elements.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="understanding-quick-start-elements_{context}"]
+= Mapping the elements in the quick start to the quick start CR
+
+This section helps you visually map parts of the quick start custom resource (CR) with where they appear in the quick start within the web console.
+
+[id="conclusion-quick-start-element_{context}"]
+== Conclusion
```

Comment:
As discussed on Slack, consider renaming each of these subsection titles to match the case of the field name, and add either "element" or "field" after. E.g.,:

```
== conclusion element
```